docs/index.qmd

```diff
@@ -1,171 +1,229 @@
-lines between A, B, C, and D) instead.
+The Ribasim QGIS plugin allows users to construct a model from scratch without programming.
+For specific tasks, like adding observed rainfall timeseries, it can be faster to use Python instead.
 
-Multiple basins may exist within the same spatial polygon, representing
-different aspects of the surface water system (perennial ditches, ephemeral
-ditches, or even surface ponding). @fig-p, @fig-s, @fig-t show the 25.0 m
-rasterized primary, secondary, and tertiary surface waters as identified by BRT
-TOP10NL [@pdoktopnl] in the Hupsel basin (as defined in the Mozart LSW's).
-These systems may represented in multiple ways.
+One can also use Ribasim Python to build entire models from base data, such that your model
+setup is fully reproducible.
 
-![Hupsel: primary surface water.](https://user-images.githubusercontent.com/13662783/187625163-d0a81bb6-7f55-4ad1-83e2-90ec1ee79740.PNG){#fig-p}
+See [Usage](https://deltares.github.io/Ribasim/core/usage.html) for more information.
 
-![Hupsel: secondary surface water.](https://user-images.githubusercontent.com/13662783/187625170-1acdfb41-7077-443f-b140-ae18cbf21e53.PNG){#fig-s}
+# Download {#sec-download}
 
-![Hupsel: tertiary surface water.](https://user-images.githubusercontent.com/13662783/187625174-3eec28b5-ddbb-4870-94c3-d9e9a43f8eb4.PNG){#fig-t}
+- Ribasim executable - Linux: [ribasim_cli_linux.zip](https://github.com/Deltares/Ribasim/releases/latest/download/ribasim_cli_linux.zip)
+- Ribasim executable - Windows: [ribasim_cli_windows.zip](https://github.com/Deltares/Ribasim/releases/latest/download/ribasim_cli_windows.zip)
+- QGIS plugin: [ribasim_qgis.zip](https://github.com/Deltares/Ribasim/releases/latest/download/ribasim_qgis.zip).
+- Generated testmodels: [generated_testmodels.zip](https://github.com/Deltares/Ribasim/releases/latest/download/generated_testmodels.zip)
 
-As a single basin (A) containing all surface water, discharging to its
-downstream basin to the west (B):
+The nightly builds contain the latest developments and can be found below. It is important to either use the release or nightly for all components.
 
-```{mermaid}
-graph LR;
-    A["basin A"] --> B["basin B"];
-```
+- Ribasim executable: [ribasim_cli.zip](https://ribasim.s3.eu-west-3.amazonaws.com/teamcity/Ribasim_Ribasim/BuildRibasimCliWindows/latest/ribasim_cli.zip).
+- QGIS plugin: [ribasim_qgis.zip](https://ribasim.s3.eu-west-3.amazonaws.com/teamcity/Ribasim_Ribasim/BuildRibasimCliWindows/latest/ribasim_qgis.zip).
 
-Such a system may be capable of representing discharge, but it cannot represent
-residence times or differences in solute concentrations: within a single basin,
-drop of water is mixed instantaneously. Instead, we may the group primary (P),
-secondary (S), and tertiary (T) surface waters. Then T may flow into S, S into
-P, and P discharges to the downstream basin (B.)
+Currently only Windows builds for `ribasim_cli.zip` are available.
```

Comment:
That is not true anymore. We also have Linux builds nowadays :)

Reply:
as verbally agreed remove reference to nightly build